tools/concepts/mfcc/test_mfcc.m

```diff
@@ -0,0 +1,219 @@
+% Script to draft MFCC (Mel frequency cepstral coefficients) audio features
+% calculating in SOF from audio stream.
+%
+% 1. Make STFT process with parameterized FFT size and hop. Hop is the number of
+%    of samples between successive FFTs. FFT lengths should be 2^N to use SOF
+%    library FFT.
+% 2. Calculate power spectrum
+% 3. Convert power spectrum to Mel bands
+% 4. Convert linear power to Decibels
+% 5. Convert each dB Mel vector to DCT (assume type-II)
+% 6. The ceptral coefficiens are output at FFT hop rate. If SOF period is shorter
+%    than hop, then every copy() does not trigger FFT. If period is longer than
+%    hop several FFTs may be computed in a single copy().
+%
+% Some ideas are from Python code example of MFCC in
+% https://www.kaggle.com/code/ilyamich/mfcc-implementation-and-tutorial/notebook
+
+% SPDX-License-Identifier: BSD-3-Clause
+%
+% Copyright (c) 2022, Intel Corporation. All rights reserved.
+
+function test_mfcc()
+
+	%% Get input signal
+	stream.fs = 48e3;
+	stream.t = 1e-3;
+	stream.channels = 1;
+	stream.frames = round(stream.fs * stream.t);
+
+	%% mfcc parameters
+	mfcc.window = 'hanning';
+	mfcc.fft_size = 1024;
+	mfcc.hop_size = 480; % number of samples between successive FFT
+	mfcc.mel_bands = 10;
```

Comment:
These are just examples to get started. Hop size could be also milliseconds in some fractional notation, not sure which is better.